ebib.el

```diff
@@ -4126,9 +4118,125 @@ special manner."
       ;; A numeric prefix argument is always non-nil when a command is called
       ;; interactively, so `pfx' can only be nil if `ebib-edit-field' is called
       ;; from Lisp code.
+      (ebib-set-field-value field result key ebib--cur-db 'overwrite (ebib-unbraced-p init-contents))
       (when pfx (ebib-next-field))
       (ebib--redisplay-field field)
-      (ebib--redisplay-index-item field))))
+      (ebib--redisplay-index-item field)
+      (ebib--set-modified t ebib--cur-db t
+			  (seq-filter (lambda (dependent)
+                                        (ebib-db-has-key key dependent))
+                                      (ebib--list-dependents ebib--cur-db))))))
+
+(defun ebib--edit-literal-field (field-name fields init-contents &optional &rest extra-tables)
+  "Edit a 'literal' type field.
+
```

Comment:
There's no need for an empty line after the first line of a doc string (here and below).

Reply:
Done